config/webpack/alias-builder.js

```diff
@@ -46,8 +46,14 @@ module.exports = {
     // Allow SPAs to provide custom module aliases.
     const moduleAliases = skyPagesConfig.skyux.moduleAliases;
     if (moduleAliases) {
+      const command = (argv && argv.command) ? argv.command : 'serve';
+
       Object.keys(moduleAliases).forEach((key) => {
-        alias[key] = spaPath(moduleAliases[key]);
+        if (command === 'serve') {
+          alias[key] = spaPath(moduleAliases[key]);
+        } else {
+          alias[key] = skyPagesConfigUtil.spaPathTemp(moduleAliases[key]);
```

Comment:
Is it true that all commands except serve should use the temp path?  Seems that perhaps we'd want only the build/build-public-library commands to use the temp path.